coderd/database/migrations/000034_remove_admin_role.up.sql

```diff
@@ -0,0 +1,20 @@
+UPDATE
+    users
+SET
+    -- Replace the role 'admin' with the role 'owner'
+    rbac_roles = array_replace(rbac_roles, 'admin', 'owner')
+WHERE
+    -- Update the first user with the role 'admin'. This should be the first
+    -- user ever, but if that user was demoted from an admin, then choose
+    -- the next best user.
+    id = (SELECT id FROM users WHERE 'admin' = ANY(rbac_roles) ORDER BY created_at ASC LIMIT 1);
```

Comment:
This could technically cause the query to fail if no rows are found, I think doing `id IN (...)` would allow it to match no rows.

Reply:
Oh interesting. Let me try it out

Reply:
@coadler It does not seem to matter.

Reply:
Hmm, I thought this would've caused an issue when you compared a non-null field to null. Sounds good then!